package.json

```diff
@@ -39,8 +39,10 @@
     "test:adapter:watch": "(cd packages/adapter && yarn run test:watch)",
     "test:airnode-abi": "(cd packages/airnode-abi && yarn run test)",
     "test:airnode-abi:watch": "(cd packages/airnode-abi && yarn run test:watch)",
-    "test:e2e": "(cd packages/node && yarn run test:e2e)",
-    "test:e2e:debug": "(cd packages/node && yarn run test:e2e:debug)",
+    "test:e2e": "yarn test:e2e-admin && yarn test:e2e-node",
+    "test:e2e-admin": "cd packages/admin && yarn run test:e2e",
+    "test:e2e-node": "cd packages/node && yarn run test:e2e",
+    "test:e2e-node:debug": "cd packages/node && yarn run test:e2e:debug",
```

Comment:
Can be useful to add the https://github.com/mysticatea/npm-run-all package to get package tests to run in parallel
```js
"test:e2e": "run-p test:e2e*,
```

Reply:
Wouldn't the output of those tests be interleaved?

Reply:
You're right. Unless the test pass you don't really care - and if they fail, you can just ran the tests in the single project.